atd-vzd/schema/ems__incidents.sql

```diff
@@ -0,0 +1,147 @@
+	mvc_form_weather text NULL,
+	pcr_additional_agencies text NULL,
+	pcr_transport_priority text NULL,
+	pcr_patient_acuity_initial text NULL,
+	pcr_patient_acuity_final text NULL,
+	unparsed_apd_incident_numbers text NULL,
+	apd_incident_numbers integer[] NULL,
+	geometry public.geometry(point, 4326) NULL,
+    austin_full_purpose bool default null,
+    location_id text default null,
+    latitude float8 default null,
+    longitude float8 default null,
+    apd_incident_number_1 integer default null,
+    apd_incident_number_2 integer default null,
+    mvc_form_date date default null,
+    mvc_form_time time default null
+);
+
+
+
+CREATE OR REPLACE FUNCTION ems_incidents_trigger()
+  RETURNS trigger AS
+$$
+BEGIN
+  update ems__incidents set
+    austin_full_purpose = (
+      select ST_Contains(jurisdiction.geometry, incidents.geometry)
+      from ems__incidents incidents
+      left join atd_jurisdictions jurisdiction on (jurisdiction.jurisdiction_label = 'AUSTIN FULL PURPOSE')
+      where incidents.id = new.id),
+    location_id = (
+      select locations.location_id
+      from ems__incidents incidents
+      join atd_txdot_locations locations on (incidents.geometry && locations.shape and ST_Contains(locations.shape, incidents.geometry))
+      where incidents.id = new.id),
+    latitude = ST_Y(ems__incidents.geometry),
+    longitude = ST_X(ems__incidents.geometry),
+    apd_incident_number_1 = ems__incidents.apd_incident_numbers[1],
+    apd_incident_number_2 = ems__incidents.apd_incident_numbers[2],
+    mvc_form_date = date(ems__incidents.mvc_form_extrication_datetime),
+    mvc_form_time = ems__incidents.mvc_form_extrication_datetime::time
+    where ems__incidents.id = new.id;
+RETURN NEW;
+END;
+$$
+LANGUAGE 'plpgsql';
+
+create trigger ems_incidents_trigger_insert
+after insert on public.ems__incidents
+for each row execute procedure ems_incidents_trigger();
+
+create trigger ems_incidents_trigger_update
+after update on public.ems__incidents
+for each row 
+WHEN (false 
+  or old.geometry IS DISTINCT FROM new.geometry
+  or old.apd_incident_numbers is distinct from new.apd_incident_numbers
+  or old.mvc_form_extrication_datetime is distinct from new.mvc_form_extrication_datetime)
+execute procedure ems_incidents_trigger();
+
+
+
+-- Column Comment
+COMMENT ON COLUMN "public"."ems__incidents"."pcr_key" IS 'Unique identifier for the patient care record in the EMS Data Warehouse. Can be used to uniquely identify records in this dataset';
+COMMENT ON COLUMN "public"."ems__incidents"."incident_date_received" IS 'The date that the incident was received by EMS. This could be the date that the EMS call taker took the call, or when it was transferred to EMS from another agency';
+COMMENT ON COLUMN "public"."ems__incidents"."incident_time_received" IS 'The time that the incident was received by EMS. This could be the time that the EMS call taker took the call, or when it was transferred to EMS from another agency';
```

Comment:
Probably need to confirm to be certain but looks like it is in local time

Reply:
It is, for sure. I bet I'm storing it in a UTC column. Thanks for pointing that out!